shell/browser/api/electron_api_download_item.cc

```diff
@@ -248,6 +249,12 @@ double DownloadItem::GetStartTime() const {
   return download_item_->GetStartTime().ToDoubleT();
 }
 
+download::DownloadInterruptReason DownloadItem::GetLastReason() const {
```

Comment:
I don't think we need to add a converter for this - instead we can call `DownloadInterruptReasonToString` directly.

```cpp
std::string DownloadItem::GetLastReason() const {
  if (!CheckAlive())
    return "NONE";
  auto reason = download_item_->GetLastReason();
  return download::DownloadInterruptReasonToString(reason);
}
```

Reply:
I don't know if I actually really like returning string that early but at the same time I don't have really strong opinion about it as can't see a case where it could kick back at us. Maybe if it would be used in more places?
Is there a specific reason why we would like to return string directly.